### PR TITLE
chore: storybook v6 part 1 - update story format 

### PR DESCRIPTION
**Pull request by SiTaggart**
- Update the story format for all stories in prep for V6
- remove unused addons

### CODE REVIEW

File: packages/paste-core/layout/stack/stories/index.stories.tsx [hunk 1]
```diff
@@ -10,95 +9,118 @@ import {Stack, StackOrientation} from '../src';
...
-      </Stack>
-    );
-  });
+// eslint-disable-next-line import/no-default-export
+export default {
+  title: 'Layout/Stack',
+  decorators: [withKnobs],
+};
+
+export const DefaultStack = (): React.ReactNode => {
```
Comment (richbachman) [inline packages/paste-core/layout/stack/stories/index.stories.tsx:18]: Should we be more deliberate about naming these exports? I noticed this one named `DefaultStack` and others just named `Default`.

File: packages/paste-core/components/breadcrumb/stories/index.stories.tsx [hunk 2]
```diff
@@ -1,39 +1,50 @@
...
+    <Breadcrumb>
+      <BreadcrumbItem href="#">Phone Numbers</BreadcrumbItem>
+      <BreadcrumbItem href="#">Active Numbers</BreadcrumbItem>
+      <BreadcrumbItem>Infrrd Germany hotline B</BreadcrumbItem>
+    </Breadcrumb>
+  );
+};
+
+AnchorsAndCurrentPage.story = {
+  name: 'Anchors and current page ',
```
Comment (richbachman) [inline packages/paste-core/components/breadcrumb/stories/index.stories.tsx:31]:
```suggestion
  name: 'Anchors and current page',
```

File: packages/paste-core/components/breadcrumb/stories/index.stories.tsx [hunk 3]
```diff
@@ -1,39 +1,50 @@
...
+          <Truncate title="Some very long text to truncate">Some very long text to truncate</Truncate>
+        </Box>
+      </BreadcrumbItem>
+      <BreadcrumbItem>Infrrd Germany hotline B</BreadcrumbItem>
+    </Breadcrumb>
+  );
+};
+
+AnchorsWithTruncation.story = {
+  name: 'Anchors with truncation ',
```
Comment (richbachman) [inline packages/paste-core/components/breadcrumb/stories/index.stories.tsx:49]:
```suggestion
  name: 'Anchors with truncation',
```

**Review by TheSisb — APPROVED** [review]
lgtm if it builds

**Pull request merged**